### DISCUSSION TITLE
Aldente Free - behavior when charge is above the limit???

### DISCUSSION THREAD

**Discussion by f41lbl0g** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#370] [Q&A]
Hello, I was wondering how aldente free behaves when the current charge of the mac is above the set limit. 

Lets say my macbook was at 55% and aldente was set to stop charge at 50%. Of course, when I plug it in, Aldente kicks in and shifts the power to the power adapter. Will it simply maintain it at 55% (when it drops to 54% back again to 55%) or will it keep letting the battery drain slowly as the battery drops due to time until my set limit at 50%, and then maintain it at 49-50?

**Comment by n-hass** [comment]
AlDente Free will hold at the current charge and solely use the power adaptor as a power source in this situation, when a limit is set below the current charge state. Activating the 'discharge' button addresses this – press it and your laptop will discharge to your limit and hold there. However, unlike AlDente Classic, this won't stay active once your Mac discharges to your set limit – 'discharge' mode will deactivate and you'll need to reactivate it again in future if you set a limit below your current charge state.
